aiidalab/app.py

```diff
@@ -320,9 +316,10 @@ def current_revision(self):
 
             return None  # current revision not on the release line
 
-    def __init__(self, name, app_data, aiidalab_apps_path, watch=True):
+    def __init__(self, name, watch=True):
```

Comment:
I believe we should keep `app_data=None, aiidalab_apps_path=None` for some time. Otherwise, the non-updated home app will fail.

Reply:
I think it is time that we properly address incompatibilites between the aiidalab system (that includes the `aiidalab` packages, but also all other packages installed into the system environment) and the dependents. Simply supporting a backwards compatible API "for some indefinite time" is simply not an option. I'll propose a plan for this.